lib/services/security/permissions.js

```diff
@@ -0,0 +1,275 @@
+const CONSTANTS = require('../..').constants;
```

Comment:
Add test for templated permissions - existing templated permissions test: https://github.com/happner/happn-3/blob/master/test/integration/security/templated-permissions.js